static/scripts/datetime/datetime.js

```diff
@@ -6,6 +6,11 @@ const FORMAT = {
 	dateTime: 'DD.MM.YYYY HH:mm',
 };
 
+const INTER_DATE_FORMAT = {
+	de: 'DD.MM.YYYY',
+	en: 'MM/DD/YYYY',
+};
+
```

Comment:
you don't need these constants here. They will be automatically retrieved from translation files.